tests/src/python/test_db_manager_sql_window.py

```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
```

Comment:
Was this removed on purpose?

Reply:
We are on Python3, so should we keep it?

Reply:
No idea, a quick internet search reveals that it might indeed be save to remove them.

In order to not mix topics: can you open a separate pull request where you remove all of them and where we can discuss about the implications?

Reply:
Ok, I have reverted